velox/docs/functions/presto/aggregate.rst

```diff
@@ -434,6 +434,41 @@ Statistical Aggregate Functions
     Returns linear regression slope of input values. ``y`` is the dependent
     value. ``x`` is the independent value.
 
+.. function:: regr_avgx(y, x) -> double
```

Comment:
Functions should appear in alphabetical order. Please, update to preserve such order.

Reply:
done